crates/sui-framework/packages/deepbook/tests/clob_tests.move

```diff
@@ -423,6 +424,11 @@ module deepbook::clob_test {
             let pool = test::take_shared<Pool<SUI, USD>>(&mut test);
             let account_cap = test::take_from_address<AccountCap>(&test, alice);
             let account_cap_user = object::id(&account_cap);
+            let (base_available, base_locked, quote_available, quote_locked) = account_balance<SUI, USD>(&pool, &account_cap);
```

Comment:
Could you add dedicated unit tests for this new functionality in both `custodian` and `clob` (rather than mixing this opaquely into an existing end-to-end test)?

Reply:
yep we can add a dedicated test to `custodian`